packages/wonder-blocks-cell/src/components/internal/cell-core.js

```diff
@@ -127,6 +184,50 @@ const styles = StyleSheet.create({
         // content is too short.
         marginLeft: "auto",
     },
+
+    /**
+     * States
+     */
+    hovered: {
```

Comment:
I think the test coverage dropped as there's no tests for some of the states here?